Dockerfile

```diff
@@ -3,7 +3,9 @@ FROM golang:1.19.3-alpine AS builder
 
 WORKDIR /app
 COPY . /app
-RUN CGO_ENABLED=0 go build -trimpath -ldflags "-s -w" -v -o atlantis .
+RUN --mount=type=cache,target=/go/pkg/mod \
```

Comment:
Can we set these args in the github actions yaml instead of in the dockerfile directly?